misk/src/main/kotlin/misk/web/WebConfig.kt

```diff
@@ -6,21 +6,44 @@ import misk.security.ssl.TrustStoreConfig
 import misk.web.exceptions.ActionExceptionLogLevelConfig
 
 data class WebConfig(
+  /** HTTP port to listen on, or 0 for any available port. */
   val port: Int,
+
+  /** If a connection is unused for this many milliseconds, it is closed. */
   val idle_timeout: Long,
+
+  /** The network interface to bind to. Null or 0.0.0.0 to bind to all interfaces. */
   val host: String? = null,
+
   val ssl: WebSslConfig? = null,
+
+  /** HTTP/2 support is currently opt-in because we can't load balance it dynamically. */
```

Comment:
`true` being the default makes HTTP/2 opt-out.

Reply:
Yep. I’m gonna flip in follow-up